mautic/mautic
# Fix form fields not loading when custom properties are defined

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✔️ 
| New feature/enhancement? (use the a.x branch)      | ❌
| Deprecations?                          | ❌
| BC breaks? (use the c.x branch)        | ❌
| Automated tests included?              | ❌ 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes issue: #13542



## Description
After Updating Mautic from 4.4.11 to 5.0.2 there is an issue with the form fields checkbox, radio and select.
If the input field is mapped with a contact/company field and custom properties are defined, the assigned contact/company fields list choices are loading.

**Custom properties**
![image](https://github.com/mautic/mautic/assets/23354033/57b79ede-8c03-432e-b4a3-9cb9061e239d)

**Output**
![image](https://github.com/mautic/mautic/assets/23354033/66e1432d-79db-471d-93ee-02701f50e960)

### How can we reproduce this issue?
Step 1: Create a form
Step 2: Add field type radio
Step 3: Connect field with the contact title field
Step 4: Define custom properties
Step 5: See form output

With this PR, the issue should be resolved and the form fields should work properly.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the PR and click the green ‘Gitpod’ button to launch the environment with the PR applied. Alternatively, manually check out the PR branch in your local setup.
3. Create or identify a Mautic entity, such as a Contact, that has custom properties. Perform actions that trigger the loading of these properties and ensure they are displayed correctly without errors.
4. Report findings.
